assets/src/edit-story/app/media/local/test/useContextValueProvider.js

```diff
@@ -0,0 +1,263 @@
+        file: 'IMAGE-768x1579.jpg',
+        width: 768,
+        height: 1579,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-768x1579.jpg',
+      },
+      '1536x1536': {
+        file: 'IMAGE-747x1536.jpg',
+        width: 747,
+        height: 1536,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-747x1536.jpg',
+      },
+      '2048x2048': {
+        file: 'IMAGE-996x2048.jpg',
+        width: 996,
+        height: 2048,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-996x2048.jpg',
+      },
+      'web-stories-poster-portrait': {
+        file: 'IMAGE-640x853.jpg',
+        width: 640,
+        height: 853,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-640x853.jpg',
+      },
+      'web-stories-poster-landscape': {
+        file: 'IMAGE-853x640.jpg',
+        width: 853,
+        height: 640,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-853x640.jpg',
+      },
+      'web-stories-poster-square': {
+        file: 'IMAGE-640x640.jpg',
+        width: 640,
+        height: 640,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-640x640.jpg',
+      },
+      'web-stories-publisher-logo': {
+        file: 'IMAGE-96x96.jpg',
+        width: 96,
+        height: 96,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-96x96.jpg',
+      },
+      'web-stories-thumbnail': {
+        file: 'IMAGE-150x308.jpg',
+        width: 150,
+        height: 308,
+        mime_type: 'image/jpeg',
+        source_url:
+          'http://wp.local/wp-content/uploads/2020/09/IMAGE-150x308.jpg',
+      },
+      full: {
+        file: 'IMAGE.jpg',
+        width: 1080,
+        height: 2220,
+        mime_type: 'image/jpeg',
+        source_url: 'http://wp.local/wp-content/uploads/2020/09/IMAGE.jpg',
+      },
+    },
+    attribution: undefined,
+  },
+];
+
+describe('useContextValueProvider', () => {
+  let reducerState;
+  let reducerActions;
+  beforeEach(() => {
+    reducerState = {
+      processing: [],
+      processed: [],
+      media: [],
+      pageToken: '',
+      mediaType: '',
+      searchTerm: '',
+    };
+    reducerActions = {
+      fetchMediaStart: jest.fn(),
+      fetchMediaSuccess: jest.fn(),
+      fetchMediaError: jest.fn(),
+      resetFilters: jest.fn(),
+      setMedia: jest.fn(),
+      setMediaType: jest.fn(),
+      setSearchTerm: jest.fn(),
+      setNextPage: jest.fn(),
+      setProcessing: jest.fn(),
+      removeProcessing: jest.fn(),
+      updateMediaElement: jest.fn(),
+      deleteMediaElement: jest.fn(),
+    };
+
+    const useUploadMediaResult = {
```

Comment:
This can be applied inline (here and below):

```
useUploadMedia.mockImplementation(() => ({
  uploadMedia: jest.fn(),
  isUploading: jest.fn(),
}));
```

Reply:
Done.